diff --git a/craftos.d.ts b/craftos.d.ts
index bf9908fa5a1ace9c10a556dd06e7b294539a959a..45244b8198341754c516d7b01d903cc72a5599f4 100755
--- a/craftos.d.ts
+++ b/craftos.d.ts
@@ -738,9 +738,9 @@ declare namespace turtle {
   function attackUp(toolSide?: string): boolean;
   function attackDown(toolSide?: string): boolean;
 
-  function dig(toolSide?: string): boolean;
-  function digUp(toolSide?: string): boolean;
-  function digDown(toolSide?: string): boolean;
+  function dig(toolSide?: string): LuaMultiReturn<[boolean, string]>;
+  function digUp(toolSide?: string): LuaMultiReturn<[boolean, string]>;
+  function digDown(toolSide?: string): LuaMultiReturn<[boolean, string]>;
 
   function place(toolSide?: string): boolean;
   function placeUp(toolSide?: string): boolean;
@@ -758,13 +758,13 @@ declare namespace turtle {
   function compareUp(toolSide?: string): boolean;
   function compareDown(toolSide?: string): boolean;
 
-  function drop(toolSide?: string): boolean;
-  function dropUp(toolSide?: string): boolean;
-  function dropDown(toolSide?: string): boolean;
+  function drop(count?: number): boolean;
+  function dropUp(count?: number): boolean;
+  function dropDown(count?: number): boolean;
 
-  function suck(toolSide?: string): boolean;
-  function suckUp(toolSide?: string): boolean;
-  function suckDown(toolSide?: string): boolean;
+  function suck(count?: number): boolean;
+  function suckUp(count?: number): boolean;
+  function suckDown(count?: number): boolean;
 
   function refuel(): boolean;
   function refuel(quantity: number): boolean;